shared/src/metabase/mbql/util.cljc

```diff
@@ -455,7 +455,10 @@
   "Is a given artihmetics clause operating on datetimes?"
```

Comment:
Agreed, it's misleading. So is the name `datetime-arithmetics?`. After this PR is merged I want to dive into this, because it's usage in the implementation of `:+` is tangled with its meaning.
`https://github.com/metabase/metabase/blob/29299781da66978a313a9452e459eb12d245ab0e/src/metabase/driver/sql/query_processor.clj#L438`